Gemfile.lock

```diff
@@ -395,6 +395,7 @@ PLATFORMS
   arm64-darwin-21
   x86_64-darwin-19
   x86_64-darwin-20
+  x86_64-darwin-21
```

Comment:
Back to your old mac?

Reply:
Don’t ask. Same Mac, but `eventmachine` on another Ruby project was moaning about ARM. Turns out you can run Terminal in Rosetta. How was your Christmas 😂